earth_enterprise/src/installer/install_server.sh

```diff
@@ -25,6 +25,7 @@ SCRIPTDIR=`dirname $0`
 . $SCRIPTDIR/common.sh
 
 # config values
+ASSET_ROOT="/gevol/assets"
```

Comment:
Server install doesn't need asset root.  Server can be installed without installing fusion.